src/components/App.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import styled, { ThemeProvider } from 'styled-components';
+import GlobalStyles from '../styles/GlobalStyles';
```

Comment:
opt* Some file imports grow to be pretty fat, so I personally like to use some semantics to 'separate them' like so;

```js
import libraries

import data

import hooks

import child components

import styles

import util
```

This is totally preference, but if you come up with your own 'separation' strategy, itll make looking over a file much easier